src/log.cpp

```diff
@@ -459,13 +459,8 @@ void log_printf(logLevel level, const char* format, ...) {
     log_file.close();
 
     /* Write the log message to the shell */
-    std::cout << msg_string;
+    printf(msg_string.c_str());
   }
-
-
-  /* If the message was a runtime error, exit the program */
-  if (level == ERROR)
-    exit(1);
 }
```

Comment:
It would still be nice for the program to stop execution after a log message with `ERROR` has been called. Have you looked into alternative ways of interrupting the program execution without destructing objects and killing the ipython kernel? I looked around but couldn't find a gracefully way to stop the execution without killing the kernel.